src/types/chain.ts

```diff
@@ -105,3 +105,11 @@ export type BookingPopupState = {
     _class: RezervoClass;
     action: BookingPopupAction;
 };
+
+export type ExcludeClassTimeFilter = {
+    weekday: number;
```

Comment:
What about using `WeekdayNumbers` type from Luxon?

Reply:
Changed to using `WeekdayNumbers`, `HourNumbers` and `MinuteNumber`, but I could not find a validation function to enforce at runtime, so relying on `1 <= weekday <= 7` `datetime.isValid`

:white_check_mark: